lib/matplotlib/contour.py

```diff
@@ -906,57 +809,9 @@ def __init__(self, ax, *args,
     allkinds = property(lambda self: [
         [subp.codes for subp in p._iter_connected_components()]
         for p in self.get_paths()])
-    tcolors = _api.deprecated("3.8")(property(lambda self: [
-        (tuple(rgba),) for rgba in self.to_rgba(self.cvalues, self.alpha)]))
-    tlinewidths = _api.deprecated("3.8")(property(lambda self: [
-        (w,) for w in self.get_linewidths()]))
     alpha = property(lambda self: self.get_alpha())
     linestyles = property(lambda self: self._orig_linestyles)
 
-    @_api.deprecated("3.8", alternative="set_antialiased or get_antialiased",
-                     addendum="Note that get_antialiased returns an array.")
-    @property
-    def antialiased(self):
-        return all(self.get_antialiased())
-
-    @antialiased.setter
-    def antialiased(self, aa):
-        self.set_antialiased(aa)
-
-    @_api.deprecated("3.8")
-    @property
-    def collections(self):
-        # On access, make oneself invisible and instead add the old-style collections
-        # (one PathCollection per level).  We do not try to further split contours into
-        # connected components as we already lost track of what pairs of contours need
-        # to be considered as single units to draw filled regions with holes.
-        if not hasattr(self, "_old_style_split_collections"):
```

Comment:
There's another check for `_old_style_split_collections` in `_split_path_and_get_label_rotation` that can be removed.